types/cbor-js/cbor-js-tests.ts

```diff
@@ -0,0 +1,5 @@
+import CBOR from "cbor-js"
```

Comment:
you would eventually need this:
```suggestion
import * as CBOR from "cbor-js"
```